app/Livewire/Security/CloudInitScriptForm.php

```
@@ -0,0 +1,101 @@
+            $this->name = $cloudInitScript->name;
+            $this->script = $cloudInitScript->script;
+        } else {
+            $this->authorize('create', CloudInitScript::class);
+        }
+    }
+
+    protected function rules(): array
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'script' => ['required', 'string', new \App\Rules\ValidCloudInitYaml],
```

PHPMD (2.15.0)

Missing class import via use statement (line '39', column '52'). (undefined)

(MissingImport)